core/src/data/clientGeoJsonSource.h

```diff
@@ -23,7 +23,8 @@ class ClientGeoJsonSource : public DataSource {
 
 public:
 
-    ClientGeoJsonSource(const std::string& _name, const std::string& _url, int32_t _maxZoom = 18);
+    ClientGeoJsonSource(const std::string& _name, const std::string& _url,
+                        int32_t _minDisplayZoom = 0, int32_t _maxDisplayZoom = INT32_MAX, int32_t _maxZoom = 18);
```

Comment:
Doesn't the map actually proxy tile higher zooms than 18? What if we wanted to zoom to 28 and explore a map of the stores in a mall or something?